docs/adr/0010-use-parameter-store-for-secrets-and-variables.md

```diff
@@ -0,0 +1,35 @@
+10. Decision to use Parameter Store for secrets and variables
+Date: 2021-06-16
+
+Status  
+In Review
+
+Context  
+Currently, we store secrets in 'credstash' which is essentially an encrypted DynamoDB table. The limitations of this system include, but are not limited to:
```

Comment:
Can we add a section describing secrets. Are these Ops only secrets or also secrets that other platform teams and possibly VFS teams store?